content/tutorial/tutorial.md

```diff
@@ -372,7 +372,7 @@ class Board extends React.Component {
   }
 
   render() {
-    const status = 'Next player: X';
+    const status = 'Siguiente jugador: X';
```

Comment:
Please, roll back this change, we are not translating any code - we're just translating code comments.